docs/cymbal-shops.md

```diff
@@ -0,0 +1,25 @@
+# Cymbal Shops Branding
+
+By default, when you deploy this sample app, the "Online Boutique" branding (logo and wording) will be used.
+But you may want to use Google Cloud's fictitious company, _Cymbal Shops_, instead.
+
+To use "Cymbal Shops" branding, set the `CYMBAL_LOGO` environment variable to `"true"` in the the Kubernetes manifest (`.yaml`) for the `frontend` Deployment.
+
+```
```

Comment:
Can we provide a command-line command that can be copy + pasted to get the immediate change?

Reply:
Great suggestion! And yes, I agree, this is a bit tricky (i.e., complex `sed` command, and need to account for development _and_ production manifests).
I shall skip.